.github/workflows/stale.yml

```diff
@@ -1,19 +1,48 @@
-name: "Mark and close stale issues"
+name: "Label and close stale issues & PRs"
+
 on:
   schedule:
   - cron: "0 0 * * *"
+  workflow_dispatch:
+    inputs:
+      debug:
+        description: "Run in debug mode (dry run)"
+        type: boolean
+        default: true
```

Comment:
IIUC, the debug mode is not in effect when this get started on cron schedule, can you please confirm?

Reply:
That's right; that setting around lines 10-11 is only for when the workflow is invoked manually, via workflow_dispatch. My thinking is that it's rare for someone to want to invoke it manually, and when they do, it would be better to err on the side of caution and not have it take action unless the user confirms they really want to. (If that doesn't make sense – and maybe it doesn't! – we should change it to false.)

Otherwise, on cron schedule, the value of the flag is set around line 20, and defaults to `false`:

```
debug-only: ${{inputs.debug || false}}
```

`inputs.debug` will be an empty string if the workflow is not invoked by workflow_dispatch.